fastlane/Fastfile

```diff
@@ -1,7 +1,6 @@
 lane :test do
-  #https://sharethemeal.org/de/checkout.html?userhash=&amount=0.4&version=2&beneficiaryUri=&beneficiarySource=contentful&thankYouUri=aqbZxjtqM0KIKUcWO6Eik
   sharethemeal(
-                  amount: "0.4",
-                  url: "https://sharethemeal.org/de/checkout.html?userhash=xxxxxx&lang=de&currency=EUR&symbol=€&amount=0.4&version=2&beneficiaryUri=40uvWurduw2QWci42KM4M&beneficiarySource=contentful&thankYouUri=aqbZxjtqM0KIKUcWO6Eik"
-               )
+    amount: "0.4",
+    url: "https://sharethemeal.org/de/checkout.html?userhash=xxxxxx&lang=de&currency=EUR&symbol=€&amount=0.4&version=2&beneficiaryUri=40uvWurduw2QWci42KM4M&beneficiarySource=contentful&thankYouUri=aqbZxjtqM0KIKUcWO6Eik"
```

Comment:
i think it is the easiest way to onboard the users - just open the stm app - hit donate button, copy url (wich opens in browser) - and use that for your fastlane setup.


otherwise users, would have to strip out specific parameters from the url - wich is kinda "mühsam".

and i don't think the flexibility of the parameters is required.